kong/conf_loader/parse.lua

```diff
@@ -432,6 +432,20 @@ local function check_and_parse(conf, opts)
         conf.ssl_dhparam = suite.dhparams
         conf.nginx_http_ssl_dhparam = suite.dhparams
         conf.nginx_stream_ssl_dhparam = suite.dhparams
+      else
+        for _, key in ipairs({
+          "nginx_http_ssl_conf_command",
```

Comment:
The only usage of those conf fields I can find is here. Could you elaborate?

Reply:
Since this set of configurations is the only one related to the security level, I believe there shouldn't be any issues.

Reply:
> Since this set of configurations is the only one related to the security level, I believe there shouldn't be any issues.

It's surprising to find that we simply ignore anything other than the SECLEVEL in this field. Could we just name it "nginx_http_ssl_security_level" or something?

Reply:
In nginx, there is no specific configuration that is solely responsible for modifying the security level.

> Nginx does not support using a single ssl_conf_command to apply multiple commands.
It is acceptable to use multiple ssl_conf_command directives in the Nginx configuration.
But it seems that kong's conf_load cannot support this unless we do special processing for these instructions.

Adding a Kong configuration as you said or making the conf_loader support array-type configurations could solve this problem. However, this would require significant changes, and there might be few users who would actually configure these settings. Therefore, this is just a most cost-effective strategy. Although I still believe that enabling conf_loader to support array-type values would provide better configurability.